docker/docker-compose.yaml

```diff
@@ -63,8 +63,30 @@ services:
     networks:
       - agglayer-localnet
 
+  zkevm-event-db:
+    container_name: zkevm-event-db
+    image: postgres:15
```

Comment:
Shall we use the `postgres:16.0-alpine` image, since it is used for all the other databases. WDYT?

Reply:
Thx! 🦅 applied